test/asynchronous/test_bulk.py

```diff
@@ -971,6 +971,9 @@ async def cause_wtimeout(self, requests, ordered):
     @async_client_context.require_replica_set
     @async_client_context.require_secondaries_count(1)
     async def test_write_concern_failure_ordered(self):
+        self.skipTest("Skipping until PYTHON-4865 is resolved.")
+        details = None
```

Comment:
What is details=None?

Reply:
To satisfy the type checker, since it is only defined in an exception block otherwise.

Reply:
ack.